packages/ua-devtools-evm-hardhat/src/tasks/oapp/config.get.ts

```diff
@@ -110,7 +110,7 @@ const action: ActionType<TaskArgs> = async ({ logLevel = 'info', oappConfig }) =
 
 task(
     TASK_LZ_OAPP_CONFIG_GET,
-    'Outputs Custom OApp Config, Default OApp Config, and Active OApp Config. Each config contains Send & Receive Librarys, Send Uln & Executor Configs, and Recieve Executor Configs',
+    'Outputs Custom OApp Config, Default OApp Config, and Active OApp Config. Each config contains Send & Receive Libraries, Send Uln & Executor Configs, and Recieve Executor Configs',
```

Comment:
There is another typo in Recieve

Reply:
Well fixed that one but as it turns out one such typo is baked into our protocol contracts

<img width="374" alt="Screenshot 2024-02-16 at 6 18 52 PM" src="https://github.com/LayerZero-Labs/devtools/assets/1451480/e191baa7-3cae-4c51-a804-e966c81d41e5">